metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/ERModelRelationProperties.pdl

```diff
@@ -0,0 +1,79 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with a ERModelRelation
+ */
+@Aspect = {
+  "name": "eRModelRelationProperties"
+}
+record ERModelRelationProperties includes CustomProperties {
+
+  /**
+   * Name of the ERModelRelation
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
+
+  /**
+  * First dataset in the ermodelrelation (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationA",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  datasetA: DatasetUrn
+
+  /**
+  * Second dataset in the ermodelrelation (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationB",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  datasetB: DatasetUrn
+
+  /**
+   * ERModelRelationFieldMapping (in future we can make it an array)
+   */
+  ermodelrelationFieldMapping: ERModelRelationFieldMapping
+
+  /**
+   * A timestamp documenting when the asset was created in the source Data Platform (not on DataHub)
+   */
+  @Searchable = {
+    "/time": {
+      "fieldName": "createdAt",
+      "fieldType": "DATETIME"
+    }
+  }
+  created: optional AuditStamp
+
+  /**
+   * A timestamp documenting when the asset was last modified in the source Data Platform (not on DataHub)
+   */
+  @Searchable = {
+    "/time": {
+      "fieldName": "lastModifiedAt",
+      "fieldType": "DATETIME"
+    }
+  }
+  lastModified: optional AuditStamp
+}
```

Comment:
let's add one more field called cardinality that looks like this:
```
/**
   * Cardinality of the relationship
   */
   cardinality: enum ERModelRelationshipCardinality {
    ONE_ONE
    ONE_N
    N_ONE
    N_N
   }
```

this field is used in acryl and is required so just want to add it even if we don't do anything with it in the UI. You'll just need to set this field when creating one of these entities in your resolver